README.md

```diff
@@ -1,8 +1,8 @@
-# CloudBees CD Add-on for AWS EKS
+# CloudBees CD/RO add-on for Amazon EKS blueprints
 
 <p align="center">
   <a href="https://www.cloudbees.com/capabilities/continuous-delivery"><img alt="cloudbees-icon" src="https://images.ctfassets.net/vtn4rfaw6n2j/7FKeUjwsXI1d2JPUIvSMZJ/be286872ace9ca3b6b66a64adbb3c16a/cb-tag-sm.svg?fm=webp&q=85" height="120px" /></a>
```

Comment:
Do we have another hosted image we can use with the new branding, to replace this one?